CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - `apphosting:secrets:grantAccess` can now grant access to emails and can grant multiple secrets at once (#8357)
 - `apphosting:secrets:set` now has flows to help with test secrets (#8359)
 - Updated `init dataconnect` templates to `v1`.
+- Fixed a bug where the emulator suite would set `FIREBASE_DATACONNECT_EMUALTOR_HOST` instead of `FIREBASE_DATA_CONNECT_EMUALTOR_HOST`.
```

Comment:
```suggestion
- Fixed a bug where the emulator suite would set `FIREBASE_DATACONNECT_EMULATOR_HOST` instead of `FIREBASE_DATA_CONNECT_EMULATOR_HOST`.
```